### PR TITLE
fix: Do not log wakeup error when app is in stopping state

### PR DESCRIPTION
**Pull request by Matovidlo**
**Changes:**
- Wait till the app is stopped and perform wakeup again, do not log the error.
- Increase context timeout to sandboxes service to 60s

-----------

### CODE REVIEW

**Review by hosekpeter — APPROVED** [review]
LGTM

File: internal/pkg/service/appsproxy/dataapps/wakeup/wakeup.go [hunk 1]
```diff
@@ -68,7 +71,7 @@ func (l *Manager) Wakeup(ctx context.Context, appID api.AppID) error {
 	item.nextRequestAfter = now.Add(Interval)
 
 	// Send the notification
-	if _, err := l.api.WakeupApp(appID).Send(ctx); err != nil {
+	if _, err := l.api.WakeupApp(appID).Send(ctx); err != nil && err.Error() != wakeupErrorToBeSkipped {
```
Comment (jachym-tousek-keboola) [inline internal/pkg/service/appsproxy/dataapps/wakeup/wakeup.go:74]: So you want to just ignore this specific error and behave as though wakeup succeeded? I don't like that tbh. Maybe just use lower log level then error instead? Also add some comment why we're treating this case differently.

**Review by jachym-tousek-keboola — CHANGES_REQUESTED** [review]

File: internal/pkg/service/appsproxy/dataapps/wakeup/wakeup.go (hunk 1 above)
Comment (Matovidlo) [reply internal/pkg/service/appsproxy/dataapps/wakeup/wakeup.go]: c971ed3496ef7dee7a7599cb1778552e53d94ee4

File: internal/pkg/service/appsproxy/dataapps/wakeup/wakeup.go [hunk 2]
```diff
@@ -68,7 +71,9 @@ func (l *Manager) Wakeup(ctx context.Context, appID api.AppID) error {
 	item.nextRequestAfter = now.Add(Interval)
 
 	// Send the notification
-	if _, err := l.api.WakeupApp(appID).Send(ctx); err != nil {
+	_, err := l.api.WakeupApp(appID).Send(ctx)
+	// If it does not succeed but app is currently stopping do not log it as error, log only other errors
```
Comment (jachym-tousek-keboola) [inline internal/pkg/service/appsproxy/dataapps/wakeup/wakeup.go:75]: I can tell as much from the code. What I want in the comment is the reason why we're doing this.

**Review by jachym-tousek-keboola — CHANGES_REQUESTED** [review]

File: internal/pkg/service/appsproxy/dataapps/wakeup/wakeup.go (hunk 2 above)
Comment (Matovidlo) [reply internal/pkg/service/appsproxy/dataapps/wakeup/wakeup.go:75]: 3979510edd27e6305524d2e04f4b316a3a8e732f

**Review by jachym-tousek-keboola — APPROVED** [review]

**Pull request merged**